packages/e2e-tests/fixtures/blocks/core__navigation.json

```diff
@@ -4,7 +4,8 @@
 		"name": "core/navigation",
 		"isValid": true,
 		"attributes": {
-			"showSubmenuIcon": true
+			"showSubmenuIcon": true,
+			"responsiveNavigation": true
```

Comment:
Nit, did you consider `hasResponsiveNavigation` to align naming with the other boolean prop used?

Reply:
Good point! I will go with `isResponsive` for this one; using "has" in this context makes it sound more like a feature toggle than a status one.